vehicle/CVehicleModelInfo.hpp

```diff
@@ -106,7 +106,9 @@ class CVehicleModelInfo : public CBaseModelInfo
     float m_min_seat_height; //0x0554
     char pad_0558[40]; //0x0558
     uint32_t m_is_jetski; //0x0580
-    char pad_0584[28]; //0x0584
-}; //Size: 0x05A0
-static_assert(sizeof(CVehicleModelInfo) == 0x5A0);
+    char pad_0584[7]; //0x0584
+    uint16_t m_ability_flag; //0x058B
```

Comment:
I have reversed the relevant native, which seems to be 588 and 58C, what is 58B

Reply:
Do you mind adding the enum of various flags to your pr, because obviously the values ​​modified by 58B and 588 are different

Reply:
It is a uint16_t. There are 2 ways you can change it. As int or as an array of byte. You can change arrays to enable parachutes or rocket boosts, etc. You don't really need enums.

Reply:
Why don’t you need enum, just like frame flags, if you modify 0x219 then enum starts from 1 << 3, if you modify 0x218 then enum starts from 1 << 11

Reply:
VEHICLE::GET_VEHICLE_HAS_PARACHUTE
![image](https://cdn.discordapp.com/attachments/963096010853347409/1100303735881736232/image.png)
VEHICLE::GET_CAR_HAS_JUMP
![image](https://cdn.discordapp.com/attachments/963096010853347409/1100306273477611520/image.png)

If you want to use 58B to cover 588 and 58C, you should provide an enum to allow others to use this offset correctly

Reply:
OK I will add them but should I add in same file?

Reply:
Yes, you can refer to here https://github.com/Yimura/GTAV-Classes/blob/57548ccd8742f0355fc674e8f4dedee119d640e5/vehicle/CCarHandlingData.hpp#L8
or add it to the YimMenu repository https://github.com/YimMenu/YimMenu/blob/master/src/core/enums.hpp

Reply:
Adding it to the same file should be fine.

eVehicleAbilityFlags perhaps?